apps/files_sharing/lib/Controller/ShareAPIController.php

```diff
@@ -549,7 +549,8 @@ public function deleteShare(string $id): DataResponse {
 	 * @param string $publicUpload If public uploading is allowed
 	 * @param string $password Password for the share
 	 * @param string|null $sendPasswordByTalk Send the password for the share over Talk
-	 * @param string $expireDate Expiry date of the share using user timezone at 00:00. It means date in UTC timezone will be used.
+	 * @param ?string $expireDate The expiry date of the share in the user's timezone (UTC) at 00:00.
```

Comment:
```suggestion
	 * @param ?string $expireDate The expiry date of the share in the user's timezone at 00:00.
```
The user’s timezone is not always UTC